Framework/Kernel/src/Atom.cpp

```diff
@@ -1,5 +1,6 @@
 #include <algorithm>
 #include <sstream>
+#include <stdexcept>
```

Comment:
No, it was unrelated (see slack #general) a few days ago. Should I remove it from this PR?